nilearn/plotting/displays.py

```diff
@@ -257,9 +257,15 @@ def transform_to_2d(self, data, affine):
                              self.direction)
         return cut
 
-    def draw_position(self, size, bg_color, **kwargs):
+    def draw_position(self, size, bg_color, decimals=False, **kwargs):
+        if decimals:
+            text = '%s=%.{}f'.format(decimals)
+            coord = float(self.coord)
+        else:
+            text = '%s=%i'
```

Comment:
not sure you need a special case for 0 decimals, `"%.0f" % coord` should give the expected result

Reply:
Will take your suggestion.